docs/platforms/unreal/index.mdx

```diff
@@ -75,6 +76,12 @@ Currently, this method is available only for C++ UE projects. Blueprint projects
 
 </Alert>
 
+<Alert>
+
+To avoid warnings during the build for licensee versions of Unreal Engine, the `EngineVersion` key is not set in the `Sentry.uplugin` for the `github` package.
```

Comment:
```suggestion
To avoid warnings during the build for license versions of Unreal Engine, the `EngineVersion` key is not set in the `Sentry.uplugin` for the `github` package.
```
I might be reading that wrong tho.